core/src/main/java/com/novoda/noplayer/model/PlayerVideoTrack.java

```diff
@@ -12,9 +12,20 @@
     private final int height;
     private final int fps;
     private final int bitrate;
-
-    @SuppressWarnings("checkstyle:ParameterNumber") // TODO group parameters into classes
-    public PlayerVideoTrack(int groupIndex, int formatIndex, String id, ContentType contentType, int width, int height, int fps, int bitrate) {
+    private final String codecName;
+    private final RendererSupport rendererCapability;
```

Comment:
This is still referencing a renderer which is an internal concept of exoplayer. `Support` implies it is only about this model and this model alone, without leaking exoplayer into our domain. Also, this variable name is still `rendererCapability`.